### PR TITLE
:cloud: move from hosting files to cloud storage

### PR DESCRIPTION
**Pull request by markysoft**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-data-etl succeeded (http://gp-data-etl-pr-37.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-data-etl succeeded (http://gp-data-etl-pr-37.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-data-etl succeeded (http://gp-data-etl-pr-37.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-data-etl succeeded (http://gp-data-etl-pr-37.dev.beta.nhschoices.net)

File: doc/architecture/decisions/0004-replace-file-hosting-with-cloud-storage.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Date: 2017-07-06
+
+## Status
+
+Accepted
+
+## Context
+
+The container host the output file for consuming applications.
```
Comment (st3v3nhunt) [inline doc/architecture/decisions/0004-replace-file-hosting-with-cloud-storage.md]: This feels a bit incomplete. Maybe `The container currently hosts...`?

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-data-etl succeeded (http://gp-data-etl-pr-37.dev.beta.nhschoices.net)

**Pull request merged**